docs/resources/equinix_fabric_gateway.md

```diff
@@ -0,0 +1,137 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "equinix_fabric_gateway Resource - terraform-provider-equinix"
+subcategory: ""
+description: |-
+  Fabric V4 API compatible resource allows creation and management of Equinix Fabric Gateway
+  ~> Note Equinix Fabric v4 resources and datasources are currently in Beta. The interfaces related to equinix_fabric_ resources and datasources may change ahead of general availability. Please, do not hesitate to report any problems that you experience by opening a new issue https://github.com/equinix/terraform-provider-equinix/issues/new?template=bug.md
+---
+
+# equinix_fabric_gateway (Resource)
+
+Fabric V4 API compatible resource allows creation and management of Equinix Fabric Gateway
```

Comment:
Same comment here about linking to the product docs or the API docs within this brief explanation.